config/deploy/grafana.hcl

```diff
@@ -0,0 +1,84 @@
+job "grafana" {
+  datacenters = ["dc1"]
+  type        = "service"
+  node_pool   = "staging"
+
+  group "grafana" {
+    count = 1
+
+    network {
+      port "grafana" {
+        to = 3000
+      }
+      dns {
+        servers = ["10.88.0.1", "128.112.129.209", "8.8.8.8", "8.8.4.4"]
```

Comment:
question: How does grafana use the google dns servers? Is this config intentional?

Reply:
It's intentional, but just a fallback. Probably not required.